File: mpp_proiect_java/src/main/java/triatlon/domain/ParticipantDTO.java
package triatlon.domain;

import java.util.List;

public class ParticipantDTO {
    private String firstName;
    private String lastName;
    private Integer punctaj;

    public String getFirstName() {
        return firstName;
    }

    public void setFirstName(String firstName) {
        this.firstName = firstName;
    }

    public String getLastName() {
        return lastName;
    }

    public void setLastName(String lastName) {
        this.lastName = lastName;
    }

    public Integer getPunctaj() {
        return punctaj;
    }

    public void setPunctaj(Integer punctaj) {
        this.punctaj = punctaj;
    }

    public ParticipantDTO(String firstName, String lastName, Integer punctaj){
        this.firstName = firstName;
        this.lastName= lastName;
        this.punctaj = punctaj;
    }
}

File: mpp_proiect_CSharp/triatlon/model/ParticipantDTO.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace triatlon.domain
{
    [Serializable]
    public class ParticipantDTO
    {
        public string firstName { get; set; }
        public string lastName { get; set; }

        public int punctaj { get; set; }
        public ParticipantDTO()
        {
           
        }
        public ParticipantDTO(string firstName1, string lastName1, int punctaj1)
        {
            this.firstName = firstName1;
            this.lastName = lastName1;
            this.punctaj = punctaj1;
        }
        public ParticipantDTO(string firstName1, string lastName1)
        {
            this.firstName = firstName1;
            this.lastName = lastName1;
        }
        
    }
}
